### ISSUE TITLE
Global shortcut to create a sticky item from the selection

### ISSUE THREAD

**Issue by AcarBurak** [pvanek/qlipper#79]
could be a fine convenience.

Thanks for this very simple, functional and reliable utility.

**Comment by Gabri** [comment]
Yes, useful, and maybe a small button on right (shown when line has focus?) to do the same directly from te list (UI)
